### DISCUSSION TITLE
Different function arguments type

### DISCUSSION THREAD

**Discussion by Tzumx** [nautechsystems/nautilus_trader#876] [Q&A]
In nautilus_trader.adapters.binance.futures.data.BinanceFuturesDataClient._request_bars we are waiting "Timestamp":
```
from_datetime: Optional[pd.Timestamp],
 to_datetime: Optional[pd.Timestamp],
```
but below in code we have
```
if from_datetime is not None:
            start_time_ms = secs_to_millis(from_datetime)
```
we are waiting for a double?
(`cpdef uint64_t secs_to_millis(double secs)`)

**Comment by cjdsellers** [comment]
Thanks for the report, seemed we fixed this in #879
